Unix/tools/allocdump

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+#
+# Script to run to collect memory allocation information for all running OMI processes
+# Note that in order for this to work:
+#   1.  Omi processes must have been built with debug turned on
+#   2.  Omi loglevel must be INFO or higher
+#   3.  Current user needs to be enabled for sudo
+
+SCRIPTDIR=$(dirname "$0")
+BINDIR=/opt/omi/bin
+GDB=gdb
+SUDO=sudo
+GDB_SCRIPT="$SCRIPTDIR/gdb_allocdump"
+
+if [ ! -f ${GDB_SCRIPT} ]; then
+    echo "Gdb script file ${GDB_SCRIPT} does not exist"
```

Comment:
Funny abbreviation for `gdb`, suggest lower-casing first letter even though it starts the line.